### PR TITLE
Pre ruby 2.5.7 Update rubocop

### PR DESCRIPTION
**Pull request by zurbergram**
## Description
Dependency needs to be updated for ruby 2.5.7 update

Main PR https://github.com/department-of-veterans-affairs/gibct-data-service/pull/579

https://github.com/department-of-veterans-affairs/va.gov-team/issues/5746

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: Rakefile [hunk 1]
```diff
@@ -6,6 +6,6 @@
 require_relative 'config/application'
 
 # Load rake support files
-Dir[Rails.root.join('lib', 'tasks', 'support', '**', '*.rb')].each { |f| require f }
+Dir[Rails.root.join('lib', 'tasks', 'support', '**', '*.rb')].sort.each { |f| require f }
```
Comment (omgitsbillryan) [inline Rakefile]: What did rubocop yell about this? J/w.

**Review by omgitsbillryan — APPROVED** [review]
LGTM

File: Rakefile (hunk 1 above)
Comment (zurbergram) [reply Rakefile]: Lint/NonDeterministicRequireOrder

```
gibct-data-service$ bundle exec rake lint
running scss-lint...
running rubocop...
.rubocop.yml: Metrics/LineLength has the wrong namespace - should be Layout
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file:
 - Style/HashEachMethods
 - Style/HashTransformKeys
 - Style/HashTransformValues
Inspecting 294 files
.W........C...........................................................................................................................................................................................................................................................................................

Offenses:

Rakefile:9:1: W: [Corrected] Lint/NonDeterministicRequireOrder: Sort files before requiring them.
Dir[Rails.root.join('lib', 'tasks', 'support', '**', '*.rb')].each { |f| require f }
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
app/controllers/uploads_controller.rb:57:34: C: [Corrected] Style/RedundantParentheses: Don't use parentheses around a method call.
    Rails.logger.error message + (error&.backtrace).to_s
                                 ^^^^^^^^^^^^^^^^^^

294 files inspected, 2 offenses detected, 2 offenses corrected

Passed. Everything looks stylish!
```

**Pull request merged**